.cloudbuild/ci/build.yaml

```diff
@@ -56,4 +56,6 @@ steps:
       - NPM_SCRIPT=build-and-package-term
       - '--build-arg'
       - YARN_FROZEN_LOCKFILE=true
+      - '--build-arg'
+      - CONNECT_TSH_BIN_PATH=skip
```

Comment:
Right, it's a magic word that I added myself. I added a check which explicitly requires you to provide a truthy value for CONNECT_TSH_BIN_PATH so that we don't ever omit this env var by mistake.

Leaving it empty should work, then I'd need to change the check from `!env.CONNECT_TSH_BIN_PATH` to `env.CONNECT_TSH_BIN_PATH === undefined` so that it permits empty strings (for example, when doing something like `CONNECT_TSH_BIN_PATH= foo`).

I just need to check if Docker will be okay with that.

https://github.com/gravitational/webapps/blob/17e14eb14140738596243fef59e98cf8658f50d3/packages/teleterm/electron-builder-config.js#L5-L28

Reply:
Chase if you think it's worth it, but otherwise feel free to ignore me.